setup.cfg

```diff
@@ -47,13 +48,14 @@ format_nongpl =
     rfc3986-validator>0.1.0
     rfc3339-validator
     uri_template
+    isoduration;python_version>'3.6'
 
 [options.entry_points]
 console_scripts =
     jsonschema = jsonschema.cli:main
 
 [options.package_data]
-jsonschema = schemas/*.json
+jsonschema = schemas/*.json, schemas/draft2020-12/*.json
```

Comment:
I have to double check, but if you see one exists, we should just use the syntax that recursively includes everything in the `schemas` directory -- we'll have more drafts there.

Reply:
I could not find a clear definition on the recursive syntax, the one I added now seems to be working.

Reply:
Recursive glob is not supported by setuptools: https://github.com/pypa/setuptools/issues/1806